src/langchain_google_bigtable/common.py

```diff
@@ -12,15 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from importlib.metadata import version
 from typing import Optional
 
 from google.cloud import bigtable
 
 default_client: Optional[bigtable.Client] = None
+PACKAGE_NAME = "langchain-google-bigtable-python"
```

Comment:
For each integrations we would like to use the format:
```
from .version import __version__

USER_AGENT_LOADER = "langchain-google-bigtable-python:document_loader" + __version__
```

Reply:
Done